app/cdap/components/RecipeDetails/index.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ *  Copyright © 2022 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Container, Drawer } from '@material-ui/core';
+import DrawerHeader from 'components/RecipeDetails/DrawerHeader';
+import RecipeDetailContainer from 'components/RecipeDetails/DetailContainer';
+
+interface IRecipeId {
```

Comment:
we do not want to write redundant code
the same interfaces are already present in DetailsContainer/index
plz remove keep them only where they are very specifically related to
and import into the other component as per the need

plz do not write redundant code

Reply:
Done